GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -256,13 +256,8 @@ public FormBrowse([NotNull] GitUICommands commands, string filter, ObjectId sele
                         }
                     }
 
-                    if (_submoduleStatusProvider.CheckSubmoduleList(status))
-                    {
-                        // Force an update
-                        _submoduleStatusUpdateNeeded = true;
-
-                        // Note: if CheckSubmoduleList() is more finegrained, this could call InitiateSubmodulesUpdate() directly
-                    }
+                    // Note: if CheckSubmoduleList() is more fine grained, this could call InitiateSubmodulesUpdate() directly
```

Comment:
This is a change, previously the code was like 
_submoduleStatusUpdateNeeded = _submoduleStatusUpdateNeeded || _submoduleStatusProvider.HasSubmodulesStatusChanged(status);

The change will set to false if update was set already for some other reason